src/components/bulk-email-tool/text-editor/TextEditor.jsx

```diff
@@ -33,9 +33,9 @@ export default function TextEditor(props) {
         height: 600,
         branding: false,
         menubar: 'edit view insert format table tools',
-        plugins: 'advlist code emoticons link lists table image language codesample',
+        plugins: 'advlist code link lists table image language codesample',
         toolbar:
-          'formatselect fontselect bold italic underline forecolor | codesample bullist numlist alignleft aligncenter alignright alignjustify indent | blockquote link emoticons image code| language',
+          'formatselect fontselect bold italic underline forecolor | codesample bullist numlist alignleft aligncenter alignright alignjustify indent | blockquote link image code| language',
```

Comment:
Dumb question... should there be a space between the pipe character ( `|`) between `code` and `language`?